node.d.plugin issue after modularizing plugins commit

```
koc:/usr/libexec/netdata/plugins.d# /usr/libexec/netdata/plugins.d/node.d.plugin debug 1 snmp
module.js:549
    throw err;
    ^

Error: Cannot find module '../../../netdata/'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/libexec/netdata/plugins.d/node.d.plugin:43:15)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```

That struck after auto-update, so I presume many installations will be affected.

--- a/collectors/node.d.plugin/node.d.plugin.in
+++ b/collectors/node.d.plugin/node.d.plugin.in
@@ -40,7 +40,7 @@ var util = require('util');
 var http = require('http');
 var path = require('path');
 var extend = require('extend');
-var netdata = require('../../../netdata');
+var netdata = require('netdata');
 
 
 // --------------------------------------------------------------------------------------------------------------------

